python/README.md

```diff
@@ -463,6 +481,23 @@ transfer_result = await sender.transfer(
 )
 ```
 
+You can also pass wait options to the `transfer` function:
```

Comment:
do we need this one given line 427 https://github.com/coinbase/cdp-sdk/pull/105/files#diff-217ed82f87b78b399e304b07a159b27dff327c0f83adf4a2fc30b03bcbf84b01R427

Reply:
removed?